services/app-graphql/src/schema.graphql

```diff
@@ -1326,6 +1326,8 @@ input UpdateContractRateInput {
 
 input UpdateDraftContractRatesInput {
     contractID: ID!
+    "date the contracts draft revision was last updated at"
```

Comment:
```suggestion
    "The updatedAt of the contract draft revision at the time this update. This is used for optimistic concurrency control"
```
@pearl-truss 
I think my description in the graphQL schema is not accurate on what this field is and what is used for. Does something like this make more sense on the naming of the field?